startpe/Cargo.toml

```diff
@@ -20,17 +20,6 @@ default = ["prefetch", "once"]
 prefetch = []
 once = ["dep:procfs"]
 
-[profile.release]
-
-codegen-units = 1
-debug = false
-debug-assertions = false
-incremental = false
-lto = "fat"
-opt-level = 's'
-panic = "abort"
-strip = "symbols"
-
```

Comment:
This shouldn't be removed since the published crate compiles startpe outside the workspace. The generated warning when compiled in the workspace is unfortunate but there's no way around it.